rootRoutes.js

```diff
@@ -6,66 +6,36 @@ const { createProxyMiddleware } = require('http-proxy-middleware');
 const applicationRoutes = require('./applicationRoutes');
 const apiRoutes = require('./apiRoutes.js');
 const countries = require('./constants/countries.js');
+const methods = require('./sendDataJson');
```

Comment:
This comment doesn't seem addressed?  I would have expected not to see `methods` anywhere in this file.

FYI, usually it's clearer to add a reply to each review comment, even if it's just "ack".

Reply:
Wait I completely missed this comment, my bad! I created the object instead of using destructing is because we need the object for jest.spyOn(object, 'method name') assignment!